docs/about/compatibility/data-type-support.rst

```diff
@@ -0,0 +1,547 @@
+.. meta::
+  :description: Supported data types in ROCm
+  :keywords: int8, float8, float8 (E4M3), float8 (E5M2), bfloat8, float16, half, bfloat16, tensorfloat32, float, float32, float64, double, AMD, ROCm, AMDGPU
+
+.. _rocm-supported-data-types:
+
+*************************************************************
+ROCm data type specifications
+*************************************************************
+
+Integral Types
+==========================================
+
```

Comment:
All tables should be introduced by preceding text. Similarly, all headers need to be followed by text before including a figure or table.

Reply:
I made the changes, asked @LisaDelaney to check the grammar.

Reply:
@MKKnorr reviewed.